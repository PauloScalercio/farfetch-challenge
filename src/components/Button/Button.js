import React from 'react';
import MaterialButton from '@material-ui/core/Button';


const Button = props => {
    return(
        <MaterialButton  size='small' {...props}>{props.children}</MaterialButton>
    );

}
export default Button;