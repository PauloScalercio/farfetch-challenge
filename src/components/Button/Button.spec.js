import React from 'react';

import Button from './index';

describe('Atom - Card', () => {
  it('should preserve structure', () => {
    const component = <Button />;
    expect(component).toMatchSnapshot();
  });
});
