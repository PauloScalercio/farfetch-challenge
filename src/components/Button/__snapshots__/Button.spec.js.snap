// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - Button should preserve structure 1`] = `<Button />`;
